bedrock/firefox/views.py

```diff
@@ -577,11 +587,24 @@ def get_template_names(self):
             else:
                 template = 'firefox/whatsnew/whatsnew-fx67.html'
         else:
-            template = 'firefox/whatsnew/index.html'
+            if show_default_sync_whatsnew(version) and ftl_file_is_active('firefox/whatsnew/whatsnew-sync'):
+                template = 'firefox/whatsnew/index-sync.html'
+            else:
+                template = 'firefox/whatsnew/index.html'
 
         # return a list to conform with original intention
         return [template]
 
+    def render_to_response(self, context, **response_kwargs):
+        templates = self.get_template_names()
+        ftl_dict = {
```

Comment:
NIT: Since this is a static dict it would be (slightly) better to defined it as a class variable instead of in this function since it'll be created once that way vs. every time in the method body. Super picky micro-optimization, but it's something.